src/Anonymization/Anonymizer/Core/IntegerAnonymizer.php

```diff
@@ -23,18 +26,67 @@ class IntegerAnonymizer extends AbstractAnonymizer
      */
     public function anonymize(Update $update): void
     {
-        if (!($this->options->has('min') && $this->options->has('max'))) {
-            throw new \InvalidArgumentException("You should provide 2 options (min and max) with this anonymizer");
+
+        if ($this->options->has('min') && $this->options->has('max')) {
+            $this->anonymizeWithMinAndMax(
+                $update,
+                (int) $this->options->get('min'),
+                (int) $this->options->get('max')
+            );
+        } elseif ($this->options->has('delta')) {
+            $this->anonymizeWithDelta($update, (int) $this->options->get('delta'));
+        } elseif ($this->options->has('percent')) {
+            $this->anonymizeWithPercent($update, (int) $this->options->get('percent'));
```

Comment:
Shouldn't we ensure that `delta` and `percent` options contain a positive integer ?

Or maybe we could just add `$delta = \abs($delta)` before using it in `anonymizeWithDelta()`? The same thing for the percent version.

Reply:
I think we should trigger an exception if percent or delta are negative. Putting such values for those options will be (most of the time) the result of a mistake or a misunderstood on how this anonymizer works.

Reply:
I agree with this solution :ok_hand: